mediadata_ai_blocklist/py/database.py

```diff
@@ -117,6 +124,36 @@ def select_all_media_houses(self):
         finally:
             cur.close()
 
+    def update_site_status(self, media_house_id, site_status, site_reachable, site_redirect, final_url):
+        try:
+            sql = """
+            UPDATE media_house
+            SET site_status = ?, site_reachable = ?, site_redirect = ?, final_url = ?
+            WHERE id = ?
+            """
+            cur = self.conn.cursor()
+            cur.execute(sql, (site_status, site_reachable,
+                        site_redirect, final_url, media_house_id))
+            self.conn.commit()
+        except Error as e:
+            print(e)
+        finally:
+            cur.close()
+
+    def get_reachable_sites(self):
+        try:
+            cur = self.conn.cursor()
+            cur.execute("SELECT * FROM media_house WHERE site_reachable = 1")
```

Comment:
Am I missing something here, doesn't `site_reachable` expect to store BOOLEAN?

Reply:
SQLite doesn't have a separate Boolean type: https://stackoverflow.com/a/580690